api/src/main/java/marquez/service/models/Job.java

```diff
@@ -41,6 +41,9 @@ public final class Job {
   @Nullable @Setter private Run latestRun;
   @Getter private final ImmutableMap<String, Object> facets;
   @Nullable private UUID currentVersion;
+  @Getter @Nullable private String jobType;
+  @Getter @Nullable private String processingType;
+  @Getter @Nullable private String integration;
```

Comment:
I would prefer we set `integration` as a label than part of the core job model. Can we hold off an exposing the `integration` type until we decide on the best way to label datasets (_redshift_, `postgres`, etc) and jobs (_spark_, _flink_, etc)?

Reply:
Yeah, let's define labels in this PR, but maybe just add a `labels` column in `jobs` (would also make sense for `datasets` in the future) and define it as string array?